mllib/src/test/scala/org/apache/spark/ml/classification/LogisticRegressionSuite.scala

```diff
@@ -234,11 +236,22 @@ class LogisticRegressionSuite extends FunSuite with MLlibTestSparkContext {
     assert(model.weights(1) ~== weightsR(1) relTol 1E-3)
     assert(model.weights(2) ~== weightsR(2) relTol 1E-3)
     assert(model.weights(3) ~== weightsR(3) relTol 1E-3)
+
+    val result = model.transform(binaryDataset)
```

Comment:
Should the block be a function that takes `model`, `weightsR`, and `binaryDataset`? It is used 9 times in this PR.